tests/scenario/test_deploy.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+from unittest.mock import Mock, PropertyMock
+
+from charms.harness_extensions.v0.evt_sequences import Event, Scenario
+from charms.observability_libs.v0.kubernetes_service_patch import KubernetesServicePatch
+from ops.model import BlockedStatus, Container, WaitingStatus
+from ops.pebble import Error as PebbleError
+from ops.pebble import PathError
+
+from charm import MimirK8SOperatorCharm
+
+
+def setup_function():
```

Comment:
TIL. Usually I `@pytest.fixture(scope=function)`.

Reply:
Seems like a better approach is:

```python
@pytest.fixture(autouse=True)
def setup():
```
`scope="function"` is redundant since [it is the default value](https://docs.pytest.org/en/stable/reference/reference.html#pytest-fixture), besides if we do not use `autouse=True` each test need to receive the fixture function as a parameter.

From my point of view this way is clean... what do you think?

Reply:
I just pushed this change so you can see it.